src/app/components/views/homepage/homepage.component.ts

```diff
@@ -17,14 +18,12 @@ export class HomepageComponent implements OnInit, OnDestroy {
   private techUUID: string;
   private jobServiceSubscription: Subscription;
   private ngUnsubscribe: Subject<void> = new Subject<void>();
-  
-  constructor(private weatherService: WeatherService, private jobService: JobService) { }
 
-  ngOnInit(): void {    
-    this.techUUID = 'mw224g'; // TODO - Make part of a sort of "login" feature. Aaron is working on this I believe, possibly a sort of modal.
-    if (!this.jobService.getResults()) { // don't call the api if it already has data
+  constructor(private weatherService: WeatherService, private jobService: JobService, public cookieService: CookieService) { }
+
+  ngOnInit(): void {
+    this.techUUID = this.cookieService.get('user');
       this.callJobServiceJobs(this.techUUID);
```

Comment:
this is tabbed in, from the deleted if wrapper. Why is that removed? I put it in so if you leave the job list and come back it won't call the API every time?
Maybe because if you log out and log back in as someone else then it won't call the api for the new user? Maybe we can save in the job service what uuid we last sent to the back end and we can re-call it if its different (like when the user logs out and logs back in). Honestly a better solution might be to figure out a way to like "reset" certain stuff whenever the user logs in, such as the job service.

Reply:
Yeah my bad, was done quickly. But you're right on the money, it was causing a bug where if you switched between users too quickly the worklist would never update, was planning on using this as a placeholder until i overhaul the whole thing when we implement auth